Yoast/javascript
# Update the mobile snippet preview.

## Summary

~~Work In Progress, pushing for some feedback. Please don't merge~~ 🙂 
[Edit]: questions have been answered, PR ready to be reviewed.

Questions:
- seems to me `#4b11a8` in the issue instruction is actually the color for _visited_ link while the one for normal links is `#1967d2`
- seems to me the description max length is already implemented (150 characters) 
- additional changes:
  - I've updated the SEO title font size to 16px (was 18)
  - updated the Date color 
  - removed the separator 

Screenshot without image and with default globe favicon:

<img width="463" alt="Screenshot 2019-05-31 at 10 19 59" src="https://user-images.githubusercontent.com/1682452/58692521-0c36f100-838f-11e9-9fdc-3e4490c01acd.png">

Screenshot with image and with custom globe favicon:

<img width="452" alt="Screenshot 2019-05-31 at 10 21 06" src="https://user-images.githubusercontent.com/1682452/58692526-10630e80-838f-11e9-8489-4fe3aa8bd868.png">

  * Package(s) involved: search-metadata-previews

  * Should the change be included in the package changelog?
  * Should the change be included in one or more plugin changelogs?
  * Package changelog item (if applicable): Updated the Google Mobile Snippet Preview
  * Plugin changelog item (if applicable): Updated the Google Mobile Snippet Preview

## Relevant technical choices:


## Test instructions

This PR can be tested by following these steps:

- pass a custom favicon e.g.: `data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAMAAAAoLQ9TAAAAmVBMVEVHcEyaHVSdA17httKkKGqkKGqkKGqkKGqkKGqkJ2qkKGqkKGr5/PXw/OXb68j///////3Snbn////u1uT+//2kKGpnrQDZ6cWhD2SjIWefGmRdqACfA2Hg28+GAEyey2rQ6rl1tRrfzM6LwUrcuc/Rp8B8uSrn6tmRAlSjV4G43ZbJg6r0/+3G5qymzniyhJ6JJFyvO33B4KXlR4xrAAAAFXRSTlMALYeqRMHHxoqNas3Oy+2Bxehf36eZkf+lAAAAo0lEQVQYlVXP1w6DMAwFUDbd29khYdMCXf//cU2Aova+2D5SFNtx5lx0fXJ+gzkeG3dp4/kqPbt2DoASwkBk6HalgYGQtp0uCWgkEwgNRCAwl0mZ5yWDyEAMpKh6JpF+AsQD0FdfFW+UNV8AIXlXqQebgbVKIZzACJGpAqO0aCYILQxLTBCYJ8R+bGZYD6t7xzq9+97CZDVes+X54e/OzW4/9x/C9xAwpoxo+QAAAABJRU5ErkJggg==`
- pass an image to be displayed in the description e.g.: `https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTUmFOm_iWPqNYy_fws7zl_66zItEuHLPAIuz5qX3Ka5wRr5tSzcTPRXfmElood_YTLlIQkEw&usqp=CAE`
- check the Mobile preview looks as intended
- check there are no visual changes in the Desktop preview
- check the "caret" icons displayed on the left when hovering the snippet parts still work correctly

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  * Meta box snippet preview (mobile)
  * check everything works including the "caret" icons on the left

## UI changes

Fixes #300
